docs/concepts/events.md

```diff
@@ -0,0 +1,119 @@
+
+- Core events
+- User-defined events
+
+Events consist of the **event name** and an optional **payload**.
+
+![Screenshot 2023-08-18 at 13.59.01.png](_events_images/Screenshot_2023-08-18_at_13.59.01.png)
+
+## Core Events
+
+Core events are events emitted directly from the FVM (Flow Virtual Machine). The events have the same name on all networks and do not follow the same naming as user-defined events (they have no address).
+
+A list of events that are emitted by the Flow network is:
+
+| Event Name                   | Description                                                             |
+| ---------------------------- | ------------------------------------------------------------------------|
+| flow.AccountCreated          | Event that is emitted when a new account gets created.                  |
+| flow.AccountKeyAdded         | Event that is emitted when a key gets added to an account.              |
+| flow.AccountKeyRemoved       | Event that is emitted when a key gets removed from an account.          |
+| flow.AccountContractAdded    | Event that is emitted when a contract gets deployed to an account.      |
+| flow.AccountContractUpdated  | Event that is emitted when a contract gets updated on an account.       |
+| flow.AccountContractRemoved  | Event that is emitted when a contract gets removed from an account.     |
+| flow.InboxValuePublished     | Event that is emitted when a Capability is published from an account.   |
+| flow.InboxValueUnpublished   | Event that is emitted when a Capability is unpublished from an account. |
+| flow.InboxValueClaimed1      | Event that is emitted when a Capability is claimed by an account.       |
+
+For more details [on the core events, you can read Cadence reference documentation](../cadence/language/core-events.md).
+
+## User-defined events
+
+Events that are defined inside contracts and when emitted follow a common naming schema. The schema consists of 4 parts: 
+
+```cadence
+A.{contract address}.{contract name}.{event type}
+```
+
+An example event would look like:
+
+![Screenshot 2023-08-18 at 14.30.36.png](_events_images/Screenshot_2023-08-18_at_14.30.36.png)
+
+The first `A` means the event is originating from a contract, which will always be the case for user-defined events. The contract address as the name implies is the location of a contract deployed on the Flow network. Next is the name of the contracted event originates from, and last is the event type defined in the contract.
+
+There are an unlimited amount of events that can be defined on Flow, but you should know about the most common ones. 
+
+### Flow Token Events
+
+The Flow Token contract is the contract that issues a core Flow token. As with any contract, it can emit events when interacted with. When we transfer the Flow token, events are emitted. You can find a lot of details on the events emitted in the [Flow Token documentation](../concepts/core-contracts/03-flow-token.md). 
```

Comment:
Recommendation to use upper case FLOW if talking specifically about the native FLOW token. The nomenclature is "Flow" talking about the chain and "FLOW" talking about the native FLOW token.